tasks/eth/zora-002-fp-upgrade/.env

```diff
@@ -0,0 +1,28 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+
+OWNER_SAFE=0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A
+COUNCIL_SAFE=0xc2819DC788505Aac350142A7A707BF9D03E3Bd03
+FOUNDATION_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
+LIVENESS_GUARD=0x24424336F04440b1c28685a38303aC33C9D14a25
+
+# No SAFE_NONCE_* values are specified, we just use the current nonces
+# SAFE_NONCE_0X5A0AAE59D09FCCBDDB6C6CCEB07B7279367C3D2A=
+# SAFE_NONCE_0XC2819DC788505AAC350142A7A707BF9D03E3BD03=
+# SAFE_NONCE_0X847B5C174615B1B7FDF770882256E2D3E95B9D92=
+
+# Expected prestate
+EXPECTED_PRESTATE=0x0354eee87a1775d96afee8977ef6d5d6bd3612b256170952a01bf1051610ee01
+
+# Expected Guardian address
+EXPECTED_GUARDIAN=0x09f7150D8c019BeF34450d6920f6B3608ceFdAf2
+
+# Contracts deployed prior to the upgrade
+EXPECTED_ANCHOR_STATE_REGISTRY_PROXY=0x7615032B5998714A056d5100f403666Be2C3AC15
+EXPECTED_ANCHOR_STATE_REGISTRY_IMPL=0x2F5c9bed66e6CD336CbbDEd66B1039682cd4fE16
+EXPECTED_PERMISSIONED_DISPUTE_GAME_IMPL=0x7541f1C32A501E13F722fBb7C57195aD4C8415E7
+EXPECTED_PERMISSIONED_DELAYED_WETH_PROXY=0xee6F71542e32930B44D6C208a15DE67E56F86087
+EXPECTED_DISPUTE_GAME_FACTORY_PROXY=0xB0F15106fa1e473Ddb39790f197275BC979Aa37e
+
+# OP Mainnet contracts for bytecode comparison
+COMPARISON_ANCHOR_STATE_REGISTRY_IMPL=0x1B5CC028A4276597C607907F24E1AC05d3852cFC
+COMPARISON_PERMISSIONED_DISPUTE_GAME_IMPL=0x91a661891248d8C4916FB4a1508492a5e2CBcb87
```

Comment:
maybe this was the old permissioned game before U13 today? I'm currently seeing 0xd113ad136a0dF1c7FfB7F1606Be1c778bCfE2f3b

Reply:
Correct, old FDG